### DISCUSSION TITLE
Will post quantum algorithms be available in the default provider?

### DISCUSSION THREAD

**Discussion by ralienpp** [openssl/openssl#25274] [Q&A] [answered]
In the light of NIST's announcement about new standards for post-quantum algorithms (FIPS 203, 204, 205), I'd like to know if there are plans to include in them in the default provider, and if so - whether there is a rough timeline for that.

Although implementations are available in third-party providers (e.g., [from OQS](https://github.com/open-quantum-safe/oqs-provider)), some projects use OpenSSL underneath and they only do so with the default provider. One example is a Python library called `cryptography` ([related discussion](https://github.com/pyca/cryptography/issues/11473)).

I did not see any information about it in the roadmap, therefore I ask about it here.

**Comment by t8m** [comment, marked as answer]
We definitely aim to include standardized post quantum algorithms in OpenSSL. There is no timeline for that feature yet.
